### ISSUE TITLE
Error message while installing: Ggeffects is corrupt

### ISSUE THREAD

**Issue by Marven82** [strengejacke/ggeffects#28]
Hi!

I tried to install sjPlot and all relevant dependencies via github (to run the syntax for plotting my graphs, which you provided earlier), using the following syntax:

library(devtools)
devtools::install_github("strengejacke/sjlabelled", force= TRUE)
devtools::install_github("strengejacke/sjmisc", force= TRUE)
devtools::install_github("strengejacke/sjstats", force= TRUE)
devtools::install_github("strengejacke/ggeffects", force= TRUE)
devtools::install_github("strengejacke/sjPlot", force= TRUE)

However, when I try to install ggeffects, I get the following error message:

Downloading GitHub repo strengejacke/ggeffects@master
from URL https://api.github.com/repos/strengejacke/ggeffects/zipball/master
Installation failed: lazy-load database 'C:/Users/Marco/Documents/R/win-library/3.5/ggeffects/R/ggeffects.rdb' is corrupt
Warning messages:
1: In eapply(ns_env(pkg), force, all.names = TRUE) :
  restarting interrupted promise evaluation
2: In eapply(ns_env(pkg), force, all.names = TRUE) :
  internal error -3 in R_decompress1

Can you please tell me how to fix this issue? Thanks in advance!

Kind regards,

Marco

**Comment by strengejacke** [comment]
Not sure why this happens. Perhaps it helps if you de-install the ggeffects-package first?

**Comment by Marven82** [comment]
Indeed; after a clean install of R and all packages it works fine. Thanks!
